albumentations/augmentations/crops/transforms.py

```diff
@@ -388,8 +420,12 @@ class RandomResizedCrop(_BaseRandomSizedCrop):
     """Torchvision's variant of crop a random part of the input and rescale it to some size.
 
     Args:
-        height (int): height after crop and resize.
-        width (int): width after crop and resize.
+        size (tuple[int] or int): Target size for the output image. Can be specified in two ways:
+            1. As a tuple (height, width) indicating the exact dimensions to resize to after cropping.
+            2. As a single integer representing the height, with the width provided as a separate argument.
+               Note: Using height and width as separate arguments is deprecated and will be removed in future versions.
```

Comment:
Let's remove `height` and `width` from docstring.

So that all new users would use only the new signature and did not even know that old one exists.

Reply:
done